web/modules/custom/mymodule/mymodule.info.yml

```diff
@@ -0,0 +1,9 @@
+name: 'My Custom Module'
```

Comment:
Please update the name of the module. 
Module name must provide information regarding the purpose of the module